Dockerfile

```diff
@@ -39,10 +39,11 @@ RUN touch                                   /usr/local/bin/vca-uninstall-package
  && echo 'apt-get -qy clean'             >> /usr/local/bin/vca-uninstall-package
 
 # Generate locales
-RUN vca-install-package apt-utils \
- && echo en_GB.UTF-8 > /etc/locale.gen \
- && vca-install-package locales
-ENV LANG=en_GB.UTF-8
+RUN vca-install-package apt-utils locales \
+ && sed -i 's/^# \(en_GB.UTF-8 UTF-8\)/\1/' /etc/locale.gen
```

Comment:
I think you need a backslash after `locale.gen`
I built it locally and get:
`Unknown instruction: &&`